src/components/card/demoBasicUsage/index.html

```diff
@@ -1,7 +1,7 @@
 <div ng-controller="AppCtrl" ng-cloak >
   <md-content class="md-padding" layout-xs="column" layout="row">
     <div flex-xs flex-gt-xs="50" layout="column">
-      <md-card>
+      <md-card md-theme="dark">
```

Comment:
Yes I like that and perhaps with multiple themes with different light/dark contrasting values